### DISCUSSION TITLE
Badge customisation

### DISCUSSION THREAD

**Discussion by Timbaron** [badges/shields#5835] [General]
How do i customize my badge?

**Comment by otilor** [comment]
https://github.com/badges/shields/discussions/5834#discussioncomment-130271

**Comment by calebcartwright** [comment]
This is starting to look like spam folks. If you have a real question please don't hesitate to ask, but please also be mindful about polluting our Discussions area and making it more difficult for Shields users to find relevant content.

**Reply by otilor** [reply]
Yeah. Sorry about that, similar questions were being asked the same time.

**Reply by calebcartwright** [reply]
I will hold off on reporting for the moment, but am going to unmark some answers as a few marked as correct were actually incorrect/incomplete and thus misleading for our users and potentially delete some as well.

Genuine questions and discussions are always welcome. Spam is not.

**Comment by calebcartwright** [comment]
Customization of badges and creating ones own badges are not the same thing. Customization of all badges (regardless of whether it's static, dynamic, or native) can be done via supported query parameters as documented on the Shields.io site

https://shields.io/#styles
